openeogeotrellis/config/config.py

```diff
@@ -245,3 +246,13 @@ class GpsBackendConfig(OpenEoBackendConfig):
     gdalinfo_python_call: bool = False
     gdalinfo_use_subprocess: bool = True  # TODO: Only keep one gdalinfo on true
     gdalinfo_use_python_subprocess: bool = False
+
+    """
+    Inject S3 profiles and tokens that allow S3 access scoped to the Job execution.      
+    """
+    provide_s3_profiles_and_tokens: bool = False
+
+    """
+    Config file to generate identity information from this Openeo Instance
+    """
+    openeo_idp_details_file: Path = Path("/opt/.idp/idp_details.json")
```

Comment:
On our infrastructure it will exist already. It is meant to be a default that would work for any environment but if for some reason in certain environments it causes issues it could be overridden. The responsibility of creating this file is however not part of the batch job driver but our infra deploy pipelines so it indeed just exists already.